src/app/main.cpp

```diff
@@ -428,6 +428,9 @@ void myMessageOutput( QtMsgType type, const char *msg )
 // The main method symbol of this library needs to be exported so it can be called by java
 APP_EXPORT
 #endif
+#ifdef WIN32
+extern "C" __declspec(dllexport)
```

Comment:
Can this be merged with the above? After all `APP_EXPORT` should do a `dllexport` but not sure about the details

    #if( ANDROID || Q_OS_WIN )